website/docs/best-practices/clone-incremental-models.md

```diff
@@ -37,14 +37,14 @@ This can be suboptimal because:
 
 You can alleviate these problems by zero copy cloning the relevant, pre-existing incremental models into your PR-specific schema as the first step of the CI job using the `dbt clone` command. This way, the incremental models already exist in the PR-specific schema when you first execute the command `dbt build --select state:modified+` so the `is_incremental` flag will be `true`. 
 
-You'll have two commands for your dbt Cloud CI check to execute:
+You'll have two commands for your <Constant name="cloud" /> CI check to execute:
 1. Clone all of the pre-existing incremental models that have been modified or are downstream of another model that has been modified:
   ```shell
-  dbt clone --select state:modified+,config.materialized:incremental,state:old
+  <Constant name="dbt" /> clone --select state:modified+,config.materialized:incremental,state:old
```

Comment:
Should we avoid this in run examples? What if somehow "dbt" changes as a product name but the execution is still `dbt run` or whatever the command may be?

Reply:
this shouldn't be here for sure! will remove

Reply:
oh this isn't expected -- the script was meant for consistency purposes and wasn't suppose to change commands because it doesn't render nicely - i've changed any commands on that have the variable though
![Screenshot 2025-04-15 at 14 51 02](https://github.com/user-attachments/assets/34a74eab-0369-4a13-a996-c7c5a989d12c)